evadb/optimizer/rules/rules_manager.py

```diff
@@ -115,7 +116,8 @@ def __init__(self, config: ConfigurationManager):
             LogicalVectorIndexScanToPhysical(),
         ]
 
-        ray_enabled = config.get_value("experimental", "ray")
+        # ray_enabled = config.get_value("experimental", "ray")
+        ray_enabled = is_ray_available()
```

Comment:
We can not turn off ray when ray is installed. Shall we do  `config.get_value("experimental", "ray") and is_ray_available()` instead?

Reply:
@gaurav274  We could probably re-enable this option. In addition to ray being installed, we can let user update this config variable to actually use `ray`. That will further reduce the likelihood of `ray` being accidentally used.